core/src/main/java/de/danoeh/antennapod/core/util/syndication/HtmlToPlainText.java

```diff
@@ -26,8 +33,40 @@
  */
 public class HtmlToPlainText {
 
+    /**
+     * Use this method to strip off HTML encoding from given text
+     * <p>
+     * Replaces bullet points with *, ignores colors/bold/...
+     *
+     * @param str String with any encoding
+     * @return Human readable text with minimal HTML formatting
+     */
+    public static String getPlainText(String str) {
+        if (!TextUtils.isEmpty(str) && isHtml(str)) {
```

Comment:
I think it is safer to explicitly return an empty string if TextUtils.isEmpty to prevent crashes at the caller when the description is null.